lib/store/application.ts

```diff
@@ -32,6 +45,35 @@ export const createApplication = createAsyncThunk(
   }
 );
 
+export const createVerifyCode = createAsyncThunk(
```

Comment:
I might not yet understand this properly - but I imagine the verification code logic should live in a separate store. This file is really meant to hold all of the logic around the application itself.

Reply:
Have moved.